### DISCUSSION TITLE
Kind request to add optional hyperparameters

### DISCUSSION THREAD

**Discussion by borankilic** [poloclub/cnn-explainer#53] [Ideas]
First of all thank you to all who participated in creating this amazing explanatory model. It really helped me getting a grasp on how convolutional neural networks work and how different layers interact with each other. Visuals and animations were meticulously prepared. I want to just recommend adding a few new hyperparameters to the "Understanding Hyperparameters" section. Adding options like dilation, groups and padding mode would  be really useful for those who are fully trying to learn the features of convolutional layers. It is just a kind recommendation you'd maybe want to consider. I really admire your work. Keep up!
